src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -3645,4 +3645,175 @@ public Long hstrlen(final String key, final String field) {
     return client.getIntegerReply();
   }
 
+  @Override
+  public EntryID xadd(final String key, final EntryID id, final Map<String, String> hash) {
```

Comment:
Is it possible to have a `EntryID.asterisk()` for this purpose?

Reply:
Added EntryID.NEW_ENTRY

We have similar for other APIs like:
EntryID.LAST_ENTRY --> "$"
EntryID.UNRECEIVED_ENTRY --> ">"